src/features/game/lib/level.ts

```diff
@@ -1,49 +1,4 @@
-import { flipObject } from "lib/utils/flipObject";
-import { getKeys } from "../types/craftables";
-
-export type BumpkinLevel =
```

Comment:
If this type was purely used to keep LEVEL_EXPERIENCE and SKILL_POINTS keys exactly the same, I can create a test that will check those objects. Or maybe it makes sense to refactor those objects in to array of {level: number, exp: number, skillPoints: number} ?

Reply:
IMO, this type is useful because it adds context to a primitive type. Changing the type to `{ [key: string]: number }` removes all context and is also confusing for some people. Even though all keys are strings under the hood, we have the ability to see these strings represented as numbers in our IDE's so it's not super clear in this instance.

The refactor to a type object might be a better solution in this case.

Reply:
I didnt find any place in code where we would need that tho. Those two objects are not referenced by key anywhere. But we had a lot of 'as BumpkinLevel' which hides any number under that type( for example '5993 as BumpkinLevel' is completely valid)